T_tent_P_its_python.py

```diff
@@ -0,0 +1,11 @@
+def get_letters() -> list:
+    """
+    A function that returns the list of all characters between a and z and between A and Z.
+    """
+    acb_list = list(map(lambda low_c: chr(low_c), range(ord('a'), ord('z') + 1)))  # adding the lower letter
```

Comment:
I would suggest looking for the easiest answer in google - like this one 
https://stackoverflow.com/questions/16060899/alphabet-range-in-python

Reply:
Amazing!